Changelog.md

```diff
@@ -47,6 +47,7 @@ Bugfixes:
  * SMTChecker: Fix internal error when deleting struct member of function type.
  * SMTChecker: Fix internal error when using user-defined types as mapping indices or struct members.
  * SMTChecker: Improved readability for large integers that are powers of two or almost powers of two in error messages.
+ * SMTChecker: Fix compiler error caused by unhandled ``z3`` expressions that come up when bitwise operators are used in solidity code.
```

Comment:
This is changelog for 0.8.18, which has already been released. Please move to 0.8.19 section.